terraform/root.tf

```diff
@@ -6,6 +6,16 @@ provider "aws" {
   }
 }
 
+provider "aws" {
+  region = "us-east-1"
```

Comment:
Is that the correct region to be using?

Reply:
Terraform said I needed to add this while it sorted out its state file or something but I've just taken it out and it works fine without it.